fluss-lake/fluss-lake-format-paimon/src/test/java/com/alibaba/fluss/lake/paimon/LakeEnabledTableCreateITCase.java

```diff
@@ -0,0 +1,406 @@
+                            org.apache.paimon.types.DataTypes.BIGINT(),
+                            org.apache.paimon.types.DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE()
+                        },
+                        new String[] {
+                            "log_c1",
+                            "log_c2",
+                            "log_c3",
+                            "log_c4",
+                            "log_c5",
+                            "log_c6",
+                            "log_c7",
+                            "log_c8",
+                            "log_c9",
+                            "log_c10",
+                            "log_c11",
+                            "log_c12",
+                            "log_c13",
+                            "log_c14",
+                            "log_c15",
+                            "log_c16",
+                            BUCKET_COLUMN_NAME,
+                            OFFSET_COLUMN_NAME,
+                            TIMESTAMP_COLUMN_NAME
+                        }),
+                null,
+                BUCKET_NUM,
+                Collections.emptyMap());
+    }
+
+    @Test
+    void testThrowExceptionWhenConflictWithSystemColumn() {
+        for (String systemColumn :
+                Arrays.asList(BUCKET_COLUMN_NAME, OFFSET_COLUMN_NAME, TIMESTAMP_COLUMN_NAME)) {
+            TableDescriptor logTable =
+                    TableDescriptor.builder()
+                            .schema(
+                                    Schema.newBuilder()
+                                            .column("log_c1", DataTypes.INT())
+                                            .column(systemColumn, DataTypes.STRING())
+                                            .build())
+                            .property(ConfigOptions.TABLE_DATALAKE_ENABLED, true)
+                            .build();
+            TablePath logTablePath = TablePath.of(DATABASE, "log_conflict_table");
+            assertThatThrownBy(() -> admin.createTable(logTablePath, logTable, false).get())
+                    .cause()
+                    .isInstanceOf(InvalidTableException.class)
+                    .hasMessage(
+                            "Column "
+                                    + systemColumn
+                                    + " conflicts with a system column name of paimon table, please rename the column.");
+        }
+    }
+
+    private void verifyPaimonTable(
+            Table paimonTable,
+            TableDescriptor flussTable,
+            RowType expectedRowType,
+            @Nullable String expectedBucketKey,
+            int bucketNum,
+            Map<String, String> expectedCustomProperties) {
+        // check pk
+        if (!paimonTable.primaryKeys().isEmpty()) {
+            assertThat(paimonTable.primaryKeys())
+                    .isEqualTo(flussTable.getSchema().getPrimaryKey().get().getColumnNames());
+        }
```

Comment:
```suggestion
        if (!flussTable.hasPrimaryKey()) {
            assertThat(paimonTable.primaryKeys()).isEmpty();
        } else {
            assertThat(paimonTable.primaryKeys())
                    .isEqualTo(flussTable.getSchema().getPrimaryKey().get().getColumnNames());
        }
```

Check no pk as well.